backend/src/backend/main.py

```diff
@@ -81,6 +82,11 @@ async def add_book(book_payload: BookAddSchema):
         zimcheck=book_payload.zimcheck,
     )
 
+    title = await Title.objects.get_or_create(
```

Comment:
You must wrap this in a try to prevent a ValueError from crashing this and return a 400 error instead